package.json

```diff
@@ -54,57 +56,49 @@
     "regenerator-runtime": "^0.13.11",
```

Comment:
do we need this still? out of scope

Reply:
Travis, I've removed this dependency. Should be fine.

Reply:
So this isn't needed as a core library in main.ts any more?

Reply:
I don't think so but I'm not too sure. I've tested and things look OK. I'll be reverting this change however and maybe we can do a bit of a cleanup later. @seeker25 Travis is that OK with you?

Reply:
I'm surprised that it built successfully since main.ts imports both this and core-js...?

Reply:
I have no further comments on this conversation.

Reply:
Travis, is it OK to resolve this conversation? @seeker25

Reply:
Sounds good to me Karim